packages/server/logging/graphqlError.ts

```diff
@@ -20,3 +21,25 @@ export const shouldLogAsInfoLevel = (err: unknown): boolean => {
 
   return err instanceof ApolloError
 }
+
+export const shouldLogAsWarnLevel = (err: unknown): boolean => {
+  if (!(err instanceof GraphQLError)) return false
+
+  if (err.message.startsWith('Cannot return null for non-nullable field')) return true
```

Comment:
i would think that query/variable validation errors have a specific error subtype/code that can just be checked